virttest/utils_test/libvirt.py

```diff
@@ -2553,6 +2554,12 @@ def set_vm_disk(vm, params, tmp_dir=None, test=None):
                      blk_source, exp_path, dist_img))
         process.run(disk_cmd, ignore_status=False)
 
+        # Change image ownership
+        if image_owner_group:
+            update_owner_cmd = ("chown %s %s/%s" %
+                                (image_owner_group, exp_path, dist_img))
+            process.run(update_owner_cmd, ignore_status=False)
```

Comment:
ignore_status is 'False' by default

Reply:
no harm to this ,even somehow make it more readable